spec/services/sirsi_response/error_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SirsiResponse::Error, type: :service do
+  describe '#initialize' do
+    subject { described_class.new error_message_raw: error_response_obj,
+                                  symphony_client: mock_client,
+                                  key: key,
+                                  session_token: session_token,
+                                  bib_type: bib_type }
+
+    let(:error) { subject }
+    let(:error_code) { 'Not a real code 1234' }
+    let(:error_message) { 'Some error message' }
+    let(:error_response_obj) {
+      { 'messageList' => [{ 'code' => error_code,
+                            'message' => error_message }] }
+    }
+    let(:mock_client) { SymphonyClient.new } # Creates actual object, stubs API responses with Sinatra
+    let(:key) { '123457' }
+    let(:session_token) { '92929292' }
+    let(:bib_type) { 'checkout' }
+
+    after do
+      Redis.current.flushall
+    end
+
+    context 'when an error for a hold occurs that does not have a translation' do
```

Comment:
```suggestion
    context 'when an error occurs that does not have a translation' do
```
since this spec is any context regardless of bib_type?

Reply:
yes yes